src/variables/utils/buildVarsOption.ts

```diff
@@ -2,8 +2,30 @@
 import {OPTION_NAME} from 'src/variables/constants'
 
 // Types
-import {File, Property} from 'src/types/ast'
-import {VariableAssignment} from 'src/types/ast'
+import {File, Property, VariableAssignment} from 'src/types'
+import {Variable} from 'src/types/variables'
+import {filterUnusedVarsBasedOnQuery} from 'src/shared/utils/filterUnusedVars'
+import {asAssignment} from '../selectors'
```

Comment:
nit: absolute path import this and import these under the respective commented sections that they match